pages/common/dig.md

```diff
@@ -13,3 +13,7 @@
 - Specify an alternate DNS server to query (8.8.8.8 is google's public DNS):
 
 `dig @8.8.8.8 {{hostname.com}}`
+
+- Perform a reverse DNS lookup on an IP address (PTR records):
```

Comment:
the expressions "an IP address" and "PTR records" seem to mismatch in number. Should they be harmonized, or am I misreading the description?